hyperspy/drawing/marker.py

```diff
@@ -77,6 +77,23 @@ def marker_properties(self, kwargs):
             except:
                 pass
 
+    def _to_dictionary(self):
+        self_dict = self.__dict__
+        marker_dict = {}
+        marker_dict['_marker_properties'] = self_dict['_marker_properties']
```

Comment:
I would probably just use `self._marker_properties`? Is there a specific reason to use `__dict__` at all?